docs/source/gbd2019_models/risk_attributable_causes/protein_energy_malnutrition/index.rst

```diff
@@ -11,13 +11,150 @@ PEM is responsible for both fatal and nonfatal outcomes within the GBD
+    - D52.8-D53.9 (other nutritional anaemias) 
+  * - Other nutritional deficiencies
+    - D64.3 (other sideroblastic anaemias)
+  * - Other nutritional deficiencies
+    - E51-E61.9 (thiamine, niacin, other B group vitamins, ascorbic acid, vitamin D, other vitamin, dietary calcium, dietary selenium, dietary zinc, and other nutrient element deficiencies)
+  * - Other nutritional deficiencies
+    - E63-E64.0 (other nutritional deficiencies and sequelae of protein-calorie malnutrition)
+  * - Other nutritional deficiencies
+    - E64.2-E64.9 (sequelae of vitamin C deficiency, rickets, other nutritional deficiencies, and unspecified nutritional deficiencies)
+  * - Other nutritional deficiencies
+    - M12.1-M12.19 (Kashin-Beck disease)
+  * - Garbage code
+    - D50, D50.0 and D50.9 (unspecified anaemia)
+
+They then run (1) an under-5 PEM model, (2) a 5-and-over PEM model, and (3) an 
+other nutritional deficiencies model. These models are scaled using CODCorrect 
+to fit the parent nutriional deficiency model. [GBD-2019-Capstone-Appendix-PEM]_
+
+Note that as PEM is defined as "a lack of dietary protein and/or energy", it 
+includes famines and severe droughts. These result in discontinuities in PEM 
+estimation, which the GBD team accounts for. The appendix specifically mentions 
+using the Tombstone report to estimate deaths due to the famine during the Great 
+Leap Forward in the 1960s in China. [GBD-2019-Capstone-Appendix-PEM]_
+
+GBD's nonfatal PEM model takes as its case definition "moderate and severe acute 
+malnutrition", defined in terms of distance from the mean WHZ score given by the 
+WHO 2006 growth standard for children. The relevant ICD 10 codes are E40-E46.9, 
+E64.0, and ICD 9 codes are 260-263.9. PEM is partitioned into the following four 
+sequelae: [GBD-2019-Capstone-Appendix-PEM]_
+
+.. list-table:: Nonfatal PEM Sequelae
+  :widths: 10 15 15 15 
+  :header-rows: 1
+  
+  * - Sequela Name
+    - WHZ range
+    - Clinical description
+    - Disability weights
+  * - Moderate wasting without oedema
+    - {WHZ_i | -3SD < WHZ_i < -2SD}
+    - Asymptomatic
+    - NA
+  * - Moderate wasting with oedema
+    - {WHZ_i | -3SD < WHZ_i < -2SD}
+    - Is very tired and irritable and has diarrhoea
+    - 0.051 (0.031–0.079)
+  * - Severe wasting without oedema
+    - {WHZ_i | WHZ_i < -3SD}
+    - Is extremely skinny and has no energy.
+    - 0.128 (0.082–0.183)
+  * - Severe wasting with oedema
+    - {WHZ_i | WHZ_i < -3SD}
+    - Is very tired and irritable and has diarrhoea. Is extremely skinny and has no energy.
+    - 0.051 (0.031–0.079); 0.128 (0.082–0.183)
+
+These are mapped onto clinically-defined wasting states as follows:
+
+.. list-table:: Clinical definitions
+  :widths: 5 10
+  :header-rows: 1
+  
+  * - Condition
+    - Estimated by GBD sequelae
+  * - Kwashiorkor
+    - {Moderate wasting with oedema} + {Severe wasting with oedema}
+  * - Marasmus
+    - {Severe wasting without oedema} + {Severe wasting with oedema}
+
```

Comment:
I think Marasmus _does not_ include severe wasting with oedema. My understanding is that anything with oedema is considered Kwashiorkor. What I think the GBD appendix means is that marasmus = severe wasting _without oedema_.

Reply:
From Dipasquale Nutrients: Edema is the distinguishing characteristic of kwashiorkor, which does not exist in marasmus [21]

Reply:
@Ninicorn maybe we should check with the modelers? The appendix says:

> We quantified non-fatal PEM burden in four mutually exclusive and
collectively exhaustive categories, reflecting distinct gradations of disability that can occur: moderate
wasting without oedema (WHZ < -2SD to < -3 SD), moderate wasting with oedema (WHZ < -2SD to < -3
SD), severe wasting without oedema (WHZ < -3SD), and severe wasting with oedema (WHZ < -3SD). The
aggregate of categories that include “oedema” can be considered equivalent to the disease state
commonly referred to as “kwashiorkor” and severe wasting can likewise be considered equivalent to
“marasmus.” For PEM, ICD 10 codes are E40-E46.9, E64.0, and ICD 9 codes are 260-263.9

Reply:
ya, let's confirm with them.

Reply:
Ok- I've asked modellers!